lnrpc/lightning.proto

```diff
@@ -1145,6 +1162,10 @@ message SendManyRequest {
 
     // Whether unconfirmed outputs should be used as inputs for the transaction.
     bool spend_unconfirmed = 8;
+
+    // CoinSelectionStrategy is the strategy to use for selecting coins
```

Comment:
This pattern of starting a comment with the name of the variable only applies to golang code btw. See the other examples here. 

So this can just start with "The coin selection strategy to use ...."

Reply:
same for other places here.